packages/steiger/src/app.ts

```diff
@@ -28,6 +20,7 @@ async function runRules({ vfs, rules }: { vfs: Folder; rules: Array<Rule> }) {
     }
 
     const ruleName = rules[ruleResultsIndex].name
+    const pluginProvidedTheRule = getPluginByRuleName(ruleName)
```

Comment:
issue (non-blocking): this reads like a sentence to me, which is a bit confusing for a variable name

suggestion: how about this:
```suggestion
    const ruleSourcePlugin = getPluginByRuleName(ruleName)
```

Reply:
Yep, I don't mind, makes sense